distro/installer/debian

```diff
@@ -1,8 +1,12 @@
 #!/bin/sh
 
+. $LKP_SRC/lib/detect-system.sh
+arch=$(get_system_arch)
+
 # Debian package installation
 export DEBIAN_FRONTEND=noninteractive
-dpkg --add-architecture i386 && apt-get update
```

Comment:
from the original logic, apt-get update is only done when --add-architecture i386 is successful. So if to keep the logic, the code can be
[ "$arch" = "x86_64" ] && dpkg --add-architecture i386 && apt-get update